pkg/metadata/metadata.go

```diff
@@ -44,6 +49,23 @@ func SetNameAndUID(obj runtime.Object, c *client.Client) error {
 	return processName(obj, c)
 }
 
+// SetLabels metadata labels for a given object.
+func SetLabels(obj runtime.Object, labels []string) error {
+	if obj == nil {
+		return ErrNilObject
+	}
+
+	for _, label := range labels {
+		kv := strings.Split(label, "=")
```

Comment:
should we use `SplitN(label, "=", 2)`, wdyt?
if so, the value could also be possible to contain other = chars too.

Reply:
Yes, that's a good idea. I checked what happens if I pass "k1=v1,k2=v2" as label to docker, and it splits at first separator and accepts the rest of the string as the value.
```
            "Labels": {
                "k1": "v1,k2=v2"
            }
```
Updating the test to not fail for this case.
Thanks!